src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php

```diff
@@ -119,6 +118,28 @@ public function GetHTML()
         return $html;
     }
 
+    protected function isRecordCreationAllowed(string $foreignTableName): bool
+    {
+        $activeUser = &TCMSUser::GetActiveUser();
```

Comment:
Assignment by reference was only required in PHP 4 - as this is new code, we should use the opportunity to remove the "&" (also in the method below).

Reply:
ok